kubetest2/internal/deployers/eksapi/node.go

```diff
@@ -177,17 +263,7 @@ func (m *nodeManager) createManagedNodegroup(infra *Infrastructure, cluster *Clu
 		},
 		AmiType: ekstypes.AMITypes(opts.AMIType),
 	}
-	if len(opts.InstanceTypes) > 0 {
-		input.InstanceTypes = opts.InstanceTypes
-	} else {
-		// managed nodegroups uses a t3.medium by default at the time of writing
-		// this only supports 17 pods, which can cause some flakes in the k8s e2e suite
-		defaultInstanceTypes, err := m.getValidDefaultInstanceTypesByEKSAMIType(input.AmiType)
-		if err != nil {
-			return fmt.Errorf("failed to get default instance types for AmiType: %s: %v", input.AmiType, err)
-		}
-		input.InstanceTypes = defaultInstanceTypes
```

Comment:
we still want `input.InstancesTypes` set right? i know the instance type resolving step was moved earlier in the process

Reply:
Yep thx! fixed